DNN Platform/Website/Providers/DataProviders/SqlDataProvider/09.12.01.SqlDataProvider

```diff
@@ -52,7 +52,24 @@ AS
 				INNER JOIN {databaseOwner}{objectQualifier}ExtensionUrlProviders P ON P.ExtensionUrlProviderID = PT.ExtensionUrlProviderID
 			WHERE   (PT.IsActive = 1)
 GO
-
+
+
+/* Fix Scheduler Frequency for Server Monitor #5634 */
+
+IF EXISTS (SELECT * 
+            FROM {databaseOwner}[{objectQualifier}Schedule] 
+            WHERE TypeFullName = 'DotNetNuke.Services.SystemHealth.WebServerMonitor, DotNetNuke' 
+            AND TimeLapse = 5 
+            AND RetryTimeLapse = 10
+            AND TimeLapseMeasurement = 'm'
+            AND RetryTimeLapseMeasurement = 'm')
+BEGIN
+    --Update the retry timelapse to 2 minutes, similar to other standard jobs
+    UPDATE {databaseOwner}[{objectQualifier}Schedule]
+    SET RetryTimeLapse = 2
+    WHERE TypeFullName = 'DotNetNuke.Services.SystemHealth.WebServerMonitor, DotNetNuke'
+END
+GO
```

Comment:
No need to query twice, and protects against the (very unlikely) scenario of having two tasks with this type which are configured differently.

```suggestion
/* Fix Scheduler Frequency for Server Monitor #5634 */

--Update the retry timelapse to 2 minutes, similar to other standard jobs
UPDATE {databaseOwner}[{objectQualifier}Schedule]
SET RetryTimeLapse = 2
WHERE TypeFullName = 'DotNetNuke.Services.SystemHealth.WebServerMonitor, DotNetNuke'
AND TimeLapse = 5 
AND RetryTimeLapse = 10
AND TimeLapseMeasurement = 'm'
AND RetryTimeLapseMeasurement = 'm';
GO
```